snuba/subscriptions/scheduler_consumer.py

```diff
@@ -149,12 +154,7 @@ def poll(self, timeout: Optional[float] = None) -> Optional[BrokerValue[Tick]]:
                 )
                 offset_interval = Interval(previous_message.offset, commit.offset)
             except InvalidRangeError:
-                logger.warning(
-                    "Could not construct valid interval between %r and %r!",
-                    previous_message,
-                    MessageDetails(commit.offset, commit.orig_message_ts),
-                    exc_info=True,
-                )
+                self.__metrics.increment("invalid_interval")
```

Comment:
Would it be valuable to have atleast some messages being logged into Sentry? If we were just seeing this metric get incremented, do we have everything needed to troubleshoot the problem?

Reply:
It won't be considered a problem anymore, as metric received time stamps are allowed to be out of order.